sessionctx/variable/sysvar.go

```diff
@@ -636,6 +636,7 @@ var defaultSysVars = []*SysVar{
 	{ScopeSession, TIDBMemQuotaNestedLoopApply, strconv.FormatInt(DefTiDBMemQuotaNestedLoopApply, 10)},
 	{ScopeSession, TiDBEnableStreaming, "0"},
 	{ScopeSession, TxnIsolationOneShot, ""},
+	{ScopeSession, TiDBEnableTablePartition, "0"},
```

Comment:
Is it safe to use a session variable to control it? I think it should be a global variable and require to restart the server.

Reply:
It's added for testing... we can't enable it before table partition is fully implemented.